week-1/Homework/mandatory/5-magic-8-ball.js

```diff
@@ -94,3 +171,196 @@ function testAll() {
 }
 
 testAll();
+
+/*https://developers.google.com/web/tools/chrome-devtools/console/*/
+
+
+
+
+/* the switch statement explained  - https://www.w3schools.com/js/js_switch.asp   The switch expression is evaluated once, then the value of the expression is compared with the values of each case. If there is a match, the associated block of code is executed. When JavaScript reaches a break keyword, it breaks out of the switch block. This will stop the execution of inside the block. It is not necessary to break the last case in a switch block. The block breaks (ends) there anyway.
+Switch cases use strict comparison (===). The values must be of the same type to match. A strict comparison can only be true if the operands are of the same type.*/
+
+// magicBall version 2 - used a switch statement and an if else if statement for the magicBall excerzise to understand switch more - two parts to a switch statement -
+//part one - function one - indexes all the possibly answers randomly
+//source - https://discuss.codecademy.com/t/help-with-javascript-8-ball-project/68618/7
+
+function shakeBall() {
+  console.log("The ball has shaken!");
+let randomNumberIndex = Math.floor(Math.random() * 20);
+console.log("The ball has shaken!");
+let eightBall = '';
+switch (randomNumberIndex) {
+  case 0:
+    eightBall = 'It is certain';
+    break;
+  case 1:
+    eightBall = 'It is decidedly so';
+    break;
+  case 2:
+    eightBall = 'Without a doubt';
+    break;
+  case 3:
+    eightBall = 'You may rely on it';
+    break;
+  case 4:
+    eightBall = 'Yes - definitely';   //veryPositive 
+    break;
+  case 5:
+    eightBall = 'As I see it, yes';
+    break;
+  case 6:
+    eightBall = 'Most likely';
+    break;
+  case 7:
+    eightBall = 'Outlook good';
+    break;
+  case 8:
+    eightBall = 'Yes';
+    break;
+  case 9:
+    eightBall = 'Signs point to yes';  //positive
+    break;
+  case 10:
+    eightBall = 'Reply hazy, try again';
+    break;
+  case 11:
+    eightBall = 'Ask again later';
+    break;
+  case 12:
+    eightBall = 'Better not tell you now';
+    break;
+  case 13:
+    eightBall = 'Cannot predict now';
+    break;
+  case 14:
+    eightBall = 'Concentrate and ask again';  // negative
+    break;
+  case 15:
+    eightBall = 'Don\'t count on it';
+    break;
+  case 16:
+    eightBall = 'My reply is no';
+    break;
+  case 17:
+    eightBall = 'My sources say no';
+    break;
+  case 18:
+    eightBall = 'Outlook not so good';
+    break;
+  case 19:
+      eightBall = 'Very doubtful';  //veryNegative
+    break;
+}
+return eightBall;
+}
+
+//part two of the switch - calls and checks the answer when the ball is shakes with either a veryPositive, positive, veryNegative, negative statement
+let answer;
+function checkAnswer(message) {
```

Comment:
Again, which of these 2 implementations do you think is best? They're both correct, so it's just down to which one you think is the most readable.